### PR TITLE
add literature: durand_1992_insitu_1977

### PR DESCRIPTION
**Pull request by TheOpenBIts**

### CODE REVIEW

File: literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  process:
+  - role: curator
+    name: Tobias Schmider
+    orcid: https://orcid.org/0000-0002-2911-2925
+    date: 2022-02-18
+    
+figure description: 
+    version: 1
+    simultaneous measurements: null 
+    comment: steady-state CV of an in air cooled Pt(110)
```
Comment (DunklesArchipel) [inline literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml:19]:
```suggestion
```

File: literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      name: NaClO4
+      type: salt
+    - concentration:
+        unit: mol / l
+        value: 0.0001
+      name: Cu
+      purity:
+        grade: 5N
+      type: salt
+      comment: Cu-ion source not provied.
```
Comment (DunklesArchipel) [inline literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml:43]: This is a special case. In the article, the source of the Cu ions is not given.
I am not sure how to handle this properly but would go and call this a salt anyway. Maybe we can assume that it is CuO and leave a comment. Another option is to find out if the authors indicated this value in other publications.
@linuxrider   any other suggestions?

File: literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - name: Ar
+      pressure:
+        unit: 1
+        value: bar
+      proportion:
+        value: 100
+      type: gas
```
Comment (DunklesArchipel) [inline literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml:50]: this is not given in the paper and thus should be omited.

File: literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    ph:
+      uncertainty: 0.1
+      value: 0.29
```
Comment (DunklesArchipel) [inline literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml:53]: see above

File: literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    temperature:
+      unit: K
+      value: 298.15
+    type: aq
+  version: 1
+source:
+  citation key: durand_1992_insitu_1977
+  url: https://doi.org/10.1016/0013-4686(92)87111-C
```
Comment (DunklesArchipel) [inline literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml:61]: I updated the DOI. Also the field is now called `url`

File: literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  electrodes:
+    configuration: three
+    reference electrode:
+      type: RHE
+    working electrode:
+      crystallographic orientation: '110'
+      geometric electrolyte contact area:
+        unit: cm-2
+      material: Pt
+      preparation procedure: Annealed in an H2-O2 flame.
```
Comment (DunklesArchipel) [inline literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml:18]: Comments such as these should always be written as a full sentence (I fixed it).

**Review by DunklesArchipel — CHANGES_REQUESTED** [review]
I'll keep this open until we resolve the issue with the unknown component.

File: literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml (hunk 2 above)
Comment (linuxrider) [reply literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml:44]: Sorry for the late reply. Do we need to obey electroneutrality in the components? If not we could have a type `ion`?
In ref 12, which is mentioned in the experimental section, recrystallized sulfate is used, maybe here the perchlorate salt is used. I would go with the new type route.

File: literature/durand_1992_insitu_1977/Durand_1992_insitu_1977.bib [hunk 7]
```diff
@@ -0,0 +1,10 @@
+@article{Durand_1992_insitu_1977,
```
Comment (DunklesArchipel) [inline literature/durand_1992_insitu_1977/Durand_1992_insitu_1977.bib:1]:
```suggestion
@article{durand_1992_insitu_1977,
```

Comment (DunklesArchipel) [reply literature/durand_1992_insitu_1977/Durand_1992_insitu_1977.bib:1]: the filename also contains capital letters

**Review by DunklesArchipel — APPROVED** [review]

**Pull request merged**